src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/visits/VisitsService.kt

```diff
@@ -67,14 +67,27 @@ class VisitsService(
               openClosedStatus = visit.visitRestriction,
             ),
           ).let { nomisResponse ->
-            VisitMappingDto(nomisId = nomisResponse.visitId, vsipId = visitBookedEvent.reference, mappingType = "ONLINE")
+            nomisResponse
+              .map {
+                VisitMappingDto(nomisId = nomisResponse.getOrThrow().visitId, vsipId = visitBookedEvent.reference, mappingType = "ONLINE")
```

Comment:
minor code improvement in next PR this should have read 
`it.visitid` rather than `nomisResponse.getOrThrow().visitId`